comps/cores/mega/orchestrator.py

```diff
@@ -25,28 +25,42 @@
 
 
 class OrchestratorMetrics:
-    # Because:
+    # Metrics use "megaservice_" name prefix, and are class variables because:
     # - CI creates several orchestrator instances
     # - Prometheus requires metrics to be singletons
```

Comment:
Here you mention singletons but on line https://github.com/opea-project/GenAIComps/blob/62e24c271e901a9219fcc2ff2a2b7cc7a07c6a8c/comps/cores/mega/orchestrator.py#L68 you are still using inc/dec for each single OrchestratorMetrics instance, right? Consider using `type(self).request_latency`?

Reply:
Functionally they are identical (at least as long as class is not subclassed).

But I decided instead to use different metric prefixes for additional Metrics/Orchestrator instances.